partner_firstname/res_user.py

```diff
@@ -25,6 +25,7 @@ class ResUsers(orm.Model):
 
     _inherit = 'res.users'
 
+    # We should use an AbstractModel to not write two time the same method
```

Comment:
Hello @jssuzanne 

I'm not sure I understood your code comment. Can you clarify?